pyproject.toml

```diff
@@ -17,10 +17,7 @@ python = ">=3.8, <3.11.9"
 bokeh = ">=2.4.2"
 dask = ">=2021.12.0"
 fastdtw = ">=0.3.4"
-fastparquet = ">=0.8.0"
```

Comment:
Why move this down? I sorted by name before...

Reply:
Just a mistake

Reply:
But it happened because I was looking at fastparquet and already I find that including both pyarrow and fastparquet is redundant. Either one is set to optional or we remove fastparquet (https://stackoverflow.com/a/77844771)

Reply:
Fastparquet is not explicitly mentioned, and pyarrow works similarly with dask, so I removed fastparquet now